dspace-server-webapp/src/test/resources/test-config.properties

```diff
@@ -14,3 +14,6 @@ test.bitstream = ./target/testing/dspace/assetstore/ConstitutionofIreland.pdf
 
 #Path for a test Taskfile for the curate script
 test.curateTaskFile = ./target/testing/dspace/assetstore/curate.txt
+
+test.scielo = ./target/testing/dspace/assetstore/scielo-test.txt
+test.sciello-single = ./target/testing/dspace/assetstore/sciello-single-record.txt
```

Comment:
If you move these files to `dspace-server-webapp/src/test/resources/org/dspace/external/provider/impl/` then these configurations are no longer needed & can be removed.